com.mattermost.Desktop.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.mattermost.Desktop",
+    "base": "io.atom.electron.BaseApp",
```

Comment:
Just to confirm did you try Electron 2?

Reply:
In first iterations I used `org.electronjs.Electron2.BaseApp` as base because I tried to copy the Slack flatpak. But then I came accross https://github.com/flathub/electron-sample-app and use this base instead. I can't remember if Electron2 worked or not. Should I try one more time? And for this build or for build directly from sources?

Reply:
You should be able to use `org.electronjs.Electron2.BaseApp` since Mattermost currently [uses](https://github.com/mattermost/desktop/blob/master/package.json#L52) that version already.

Reply:
For context the Electron2 is just a smaller version because it doesn't need libraries like GTK2. So you should always use it if you don't need the older libs.

Reply:
I tried and I think that I need `io.atom.electron.BaseApp`. With `org.electronjs.Electron2.BaseApp` I get following error:
```
/app/main/mattermost-desktop: error while loading shared libraries: libgtk-x11-2.0.so.0: cannot open shared object file: No such file or directory
```